src/ios/HealthKit.m

```diff
@@ -860,95 +874,104 @@ - (void) querySampleType:(CDVInvokedUrlCommand*)command {
   }];
 }
 
-- (void) queryCorrelationType:(CDVInvokedUrlCommand*)command {
```

Comment:
Looks like you accidentally removed `queryCorrelationType`, can you put it back so I can merge this PR?